Installer/data/IntelBluetoothFirmware/IntelBluetoothInjector.kext.MaxKernel.txt

```diff
@@ -0,0 +1 @@
+20.9.9
```

Comment:
Did you mean 21.9.99?

Reply:
Nope. This kext must not be used in Monterey(>=21.x.x). But is needed in Big sur (<= 20.99.99)